manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl

```

Comment:
```suggestion
Publisher: Hermann Schinagl
```
This must match Control Panel / Apps&Features
![image](https://user-images.githubusercontent.com/88161975/204129763-5b11716e-e113-4b81-99e5-c643fef4c123.png)